src/main/java/Duke.java

```diff
@@ -1,10 +1,27 @@
+import java.util.Objects;
+import java.util.Scanner;
+
+/**
+ * This program is a chatbot that creates a task list for you and is able to mark tasks as done.
+ */
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String inData = "yo";
```

Comment:
Could you perhaps explain to me what this is supposed to be? I don't quite understand this part. 

EDIT: Ahh... I get it now... But would it be more accurate to change it to a final variable `START_COMMAND` instead? Since it is passed into a method `executeCommand`.